crates/wit-encoder/src/render.rs

```diff
@@ -0,0 +1,50 @@
+use std::fmt;
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct RenderOpts {
```

Comment:
Some docs explaining what this does would probably be helpful. It is not immediately clear why there are `RenderOpts`, why we have a `Render` trait rather than use `Display`, etc.

Reply:
Display is implemented for package.
Render/RenderOpts is mostly for internal use, but figured I'll make it public as we might wanna add in some options later.
I can make it private if you think it's confusing.

Reply:
@yoshuawuyts think I should make the render stuff private?